src/L.Control.Locate.js

```diff
@@ -153,6 +153,9 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
 
   const LocateControl = L.Control.extend({
     options: {
+      /** function([lat,lng]) to be called when location is found.
+      if this is set, stop() is called when location is first found. */
```

Comment:
Why would be call `stop()` when this callback is set? Maybe explain what the purpose of this API is.

Reply:
because i wanted it to be oneshot (just find my location and quit). either, the handler remains active and the callback would be called again and again until, indeed, stop() is called.  i documented this in the readme, and renamed callback to "oneshot_callback"

Reply:
So why can this not be done from outside the plugin? You can listen to the location found event on the map I think.